src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -3738,6 +3738,73 @@ bool AdaptDispatch::DoFinalTermAction(const std::wstring_view string)
     // modify the state of that mark as we go.
     return false;
 }
+// Method Description:
+// - Performs a XtermJs action
+// - Ascribes to the ITermDispatch interface
+// - Currently, the actions we support are:
+//   * Completions: An experimental protocol for passing shell completion
+//     information from the shell to the terminal. This sequence is still under
+//     active development, and subject to change.
```

Comment:
Are we using the same protocol as xterm.js here in the interim? Should we be camping their OSC like this if not?

Reply:
Also, Daniel let us know that it's a VSCode thing and not an xtermjs thing